source/extensions/filters/http/common/compressor/compressor.cc

```diff
@@ -215,13 +223,19 @@ void CompressorFilter::setDecoderFilterCallbacks(Http::StreamDecoderFilterCallba
                           StreamInfo::FilterState::StateType::Mutable);
   }
 }
-
 Http::FilterHeadersStatus CompressorFilter::encodeHeaders(Http::ResponseHeaderMap& headers,
```

Comment:
super nit, it'd be nice to reinstate the whitespace here

Reply:
sure. surprised that fix_format didn't pick it up